packages/engine/src/interaction/behaviors/interactRaycast.ts

```diff
@@ -64,22 +63,23 @@ export const interactRaycast:Behavior = (entity: Entity, { interactive }:Interac
     // console.warn('!character.viewVector')
     return;
   }
+
   const raycaster = new Raycaster();
-  let object, rayOrigin, rayDirection;
+  let object, rayOrigin, rayDirection,rayCamera,rayMouse;
   // - added mouse raycaster
-  const rayCamera = Engine.camera.clone();
-  const rayMouse = mouseScreen.normalize();
+  rayCamera = Engine.camera.clone();
+  rayMouse = mouseScreen;
```

Comment:
i think it's better to leave them both "const" since they not reassigned anywhere.